home-application/pom.xml

```diff
@@ -84,6 +85,16 @@
             <artifactId>quartz-actuator</artifactId>
             <version>${quartz-actuator.version}</version>
         </dependency>
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <version>${javax-version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
Why did we add it?

Reply:
You're right. It is not used. I delete it.